src/bin/poudriere-options.8

```diff
@@ -62,25 +62,25 @@ This command accepts the following options:
 Specifies which
 .Ar architecture
 of FreeBSD to use to show options.
-(Default: the one of jail
-.Ar jailname
-)
-.It Fl c
-Use 'config' target, which will always show the dialog for the given ports.
+.Po Default: the one of jail
+.Ar jailname Pc
 .It Fl C
 Use 'config-conditional' target, which will only bring up the dialog on new options for the given ports.
 (This is the default)
+.It Fl c
+Use 'config' target, which will always show the dialog for the given ports.
 .It Fl f Ar file
 Absolute path to a file which contains the list of ports to configure.
 Ports must be specified in the form
 .Ar category/port
 and shell-style comments are allowed.
 .It Fl j Ar jailname
 If given, configure the options only for the given jail.
-.It Fl p Ar portstree
-Run the configuration inside the given ports tree (by default uses default)
 .It Fl n
 Do not be recursive
+.It Fl p Ar portstree
+Run the configuration inside the given ports tree.
```

Comment:
Might be worth mentioning that any of `-z`, `-p`, and `-j`, used will cause the options to be saved to a specific directory for those flags.
#583 covers the issue with "using" versus "binding" for each of these. Currently they bind.